eslint.config.js

```diff
@@ -37,4 +39,11 @@ module.exports = [
             "eslint-plugin/no-unused-message-ids": 0,
         },
     },
+    {
+        // these messageIds were used outside
+        files: ["lib/**/*.js"],
```

Comment:
```suggestion
```
this should be applied to all js files?

Reply:
I didnt want to mess with the test fixtures :see_no_evil: you're right though!

Reply:
ignore: ["fixtures/"]?